.test/meta-commands/out.sh

```diff
@@ -137,6 +137,29 @@ jq -s '
 	)
 ' temp/index.json > temp/index.json.new
 mv temp/index.json.new temp/index.json
+SOURCE_DATE_EPOCH=1709081058 \
+	docker buildx build --progress=plain \
+	--provenance=false \
+	--build-context "base:latest=oci-layout://$PWD/temp/@$(jq -r '.manifests[0].digest' index.json)" \
+	--sbom=generator="$BASHBREW_BUILDKIT_SBOM_GENERATOR" \
+	--output 'type=oci,tar=false,dest=temp_sbom/' \
+	- <<<'FROM base'
+newIndexDigest="$(jq -r '.manifests[0].digest | sub("sha256:";"")' temp_sbom/index.json)"
+sbomManifestDigest="$(jq -r '.manifests[] | select(.platform.os == "unknown").digest | sub("sha256:";"")' temp_sbom/blobs/sha256/$newIndexDigest)"
+sbomManifest="$(jq -r --arg digest "sha256:$sbomManifestDigest" '.manifests[] | select(.digest == $digest)' temp_sbom/blobs/sha256/$newIndexDigest)"
+sbomDigest="$(jq -r '.layers[0].digest | sub("sha256:";"")' temp_sbom/blobs/sha256/$sbomManifestDigest)"
+cp temp_sbom/blobs/sha256/$sbomDigest temp/blobs/sha256
+cp temp_sbom/blobs/sha256/$sbomManifestDigest temp/blobs/sha256
```

Comment:
Since the way we're building leads to `"subject": []` (and thus _should_ have no references to our intermediate/incorrect image manifest digest/config), this is fine for now, but we probably need a `TODO` in here somewhere about how in the future we'll need to munge the object so it has a proper `subject` list inside the SBOM object, which means a new checksum and slightly more complicated code here (and hopefully we can write that in such a way that it's not _too_ reliant on details of the v0.1 format, which I think we can, but I don't think we need to go to that level right away).  We can take some solace in the fact that building in this way _does_ get us appropriate layer blobs references in the SBOM. :+1: